src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs

```diff
@@ -244,6 +245,24 @@ static bool IsIncorrectDotNetObjectRefUse(JsonElement item, Type parameterType)
             }
         }
 
+        /// <summary>
+        /// Receives notification that a call from .NET to JS has finished, marking the
+        /// associated <see cref="Task"/> as completed.
+        /// </summary>
+        /// <param name="arguments">The serialized arguments for the callback completion.</param>
+        public static void EndInvoke(string arguments)
+        {
+            // We don't need to notify the client if a JS interop call fails to be processed on the server.
+            // The client needs to catch potential exceptions. We do it this way so that we can log errors
+            // processing the callback completion.
```

Comment:
This comment block is phrased strangely, because it's phrased in terms of server-side Blazor, which this library isn't meant to be coupled to.

Could this be rephrased in terms of .NET/JS interop?

Reply:
I rephrased it inside the comments